bigbluebutton-html5/imports/startup/mapToAcl.js

```diff
@@ -0,0 +1,39 @@
+import Acl from '/imports/startup/acl';
+import { Meteor } from 'meteor/meteor';
+import Logger from '/imports/startup/server/logger';
+
+const emptyCollection = new Mongo.Collection("emptyCollection");
```

Comment:
Yes, is needed.

You can't return null, neither undefined inside a `publish`. you MUST return an cursor, there were only 2 options available.

1 - Call  `this.ready()`, which automatic return an empty cursor, but for me was harder to understand what was happening

2 - Return an empty cursor: I could use a `.find()` in with params on any collections,

` Users.find({"alalal":"adasdas"}) ` but again, for readability wasn't good.